src/components/vm/overview/cpuModal.jsx

```diff
@@ -250,7 +250,7 @@ export const CPUModal = ({ vm, maxVcpu, models }) => {
             <FormGroup fieldId="machines-vcpu-count-field" label={_("vCPU count")}
                        labelIcon={
                            <Popover bodyContent={_("Fewer than the maximum number of virtual CPUs should be enabled.")}>
-                               <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                               <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1088-20230704-092041-0b751cdc-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)